VulnerableApplication/Controllers/HomeController.cs

```diff
@@ -8,6 +8,9 @@ namespace VulnerableApplication.Controllers
 {
     public class HomeController : Controller
     {
+        public static string SonarPassword = "foo";
+        public static string GitHubPassword = "bar";
```

Comment:
[VULNERABILITY] 'password' detected in this expression, review this potentially hardcoded credential.